### PR TITLE
fix: [7.1.0] remove unicode chars from utf8 token symbol

### PR DESCRIPTION
**Pull request by tommasini**
**Description**
On this patch, we address unicode representing null and the end of text from a utf8 representing the token symbol.

**Screenshots/Recordings**
https://recordit.co/FfehuLYm82

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: patches/@metamask+assets-controllers+5.0.0.patch [hunk 1]
```diff
@@ -93,10 +92,19 @@ index 9ddbc28..2d12e33 100644
 +                // TODO - Remove patch once app is upgraded to React Native 0.71.6
 +                const decoded = ethers_project_abi.abiCoder.decode(['string'], result)[0];
```
Comment (Cal-L) [inline patches/@metamask+assets-controllers+5.0.0.patch:93]: I noticed these lines changed but was part of the last patch. Did you remove node modules, run yarn, run yarn patch-package, make the new change, then run yarn patch-package LIBTOPATCH?

Comment (tommasini) [reply patches/@metamask+assets-controllers+5.0.0.patch]: Good catch! fixed!

**Review by Cal-L — COMMENTED** [review]
Lgtm

**Review by Cal-L — APPROVED** [review]
Lgtm

**Review by jpcloureiro — COMMENTED** [review]
Created a [new issue](https://github.com/MetaMask/mobile-planning/issues/1009) to track the removal of those TODOs

Should we also remove this patch once the app gets React Native 0.71.6 upgrade?

**Comment by chrisleewilcox** [comment]
Works from Pixel 5 Android 12 simulator and iOS simulator.

**Review by chrisleewilcox — APPROVED** [review]
LGTM

**Pull request merged**